app/controllers/partners_controller.rb

```diff
@@ -164,6 +165,13 @@ def reactivate
 
   private
 
+  def validate_user_role
+    if current_user.kind == 'partner'
```

Comment:
This won't work, because a user can have roles both as a partner and an org admin. You need to check `current_role` to determine what the user is currently logged in as, and base it off of that.

Reply:
Ahhh got ya. Change has been made along with 2 changes from the ruby linter. Should be good to go now.